### DISCUSSION TITLE
Set Up Cloud Functions

### DISCUSSION THREAD

**Discussion by amp1016** [AntlerVC/firetable#1249] [Support / Q&A] [answered]
Hi there, I just created a **Rowy** project, I followed all the steps in your tutorials. And I am stuck in creating a table by using a template. Each time I want to use a template, a popup window informs I need to set up Cloud Functions to use the template.  But, after following the wizard instructions at the end, it is not triggering or finish the cloud function activation. And it prompts me to do the same action again and again,... (Also, my Firebase account is in the Blaze plan) See attached files. So, what do I need to do?
Thanks
AP1016

![whatsapp-ai-fe76f-•-Rowy 01](https://user-images.githubusercontent.com/132933823/236886068-b22c703b-308c-41c7-98f6-fb2aebf82cef.png)
![Project-Settings-•-whatsapp-ai-fe76f-•-Rowy 02](https://user-images.githubusercontent.com/132933823/236886076-4b2f52b4-d771-4849-aaad-63f513abdb6c.png)
![Project-Settings-•-whatsapp-ai-fe76f-•-Rowy 03](https://user-images.githubusercontent.com/132933823/236886088-3c7ada81-efb9-4ab5-8972-8d4017ec00f5.png)
![Project-Settings-•-whatsapp-ai-fe76f-•-Rowy 04](https://user-images.githubusercontent.com/132933823/236886113-ecaed361-722d-44f4-80b1-89b68fc4d863.png)

**Comment by harinij** [comment]
Hi @amp1016 - can you try the following and see if that resolves it?
- Go to the developer console on your browser and clear the site data cache or use icon as shown below to empty the cache for the site and hard restart 
![image](https://user-images.githubusercontent.com/307298/236892035-c1a21fa7-ae35-46e4-abe2-4ab4750dc269.png)
- Go to rowy.app starting URL (and not a direct table URL) and then login 
- Now if you navigate to you workspace and project and try again it should indicate cloud run is installed and template should proceed.

**Reply by amp1016** [reply]
Hey @harinij - I did the Empty cache and hard reload, and log in again from the starting URL; **but the issue remains**. Once in a selected template, I am redirected to set up Cloud Functions, and the same loop described is happening.

**Comment by amp1016** [comment]
Hey @harinij, I am stuck with the **Cloud Functions**. I did whatever you suggested, and it didn't work. Also, I tried to go around, copy the template set from the demo, and import those to enable the desired table; but nothing worked because of the Cloud Functions. So, let me know what else you can do to support and troubleshoot Rowy users.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @amp1016, could you try following the steps as described here: https://github.com/rowyio/rowy/discussions/1087#discussioncomment-4756550. Let us know if that helps! :)

**Reply by amp1016** [reply]
@BeeBombshell, That solved the problem! Great, It works. Thanks
